src/ui/ActiveScripts/ScriptProduction.tsx

```diff
@@ -41,7 +41,7 @@ export function ScriptProduction(): React.ReactElement {
       <TableBody>
         <TableRow>
           <TableCell component="th" scope="row" classes={{ root: classes.cell }}>
-            <Typography variant="body2">Total production:</Typography>
+            <Typography variant="body2">Total production since last Augment Installation:</Typography>
```

Comment:
For consistency, you should say "since last Augmentation" instead of "since last Augment Installation" - I don't think the latter phrasing is used anywhere in the game
```suggestion
            <Typography variant="body2">Total production since last Augmentation:</Typography>
```

Reply:
Oops I pulled that right from the stats page on that one. Either way whichever works. And thanks for the double checks on that stuff

Reply:
🤔  I don't know, how it's done in english, but it had to get written in lowercase? .... just some silly minds from a german 😏